requirements/tests.txt

```diff
@@ -1,4 +1,10 @@
+atomicwrites==1.3.0
```

Comment:
I don't understand why we need these imports but since they are in test requirements I am fine with them.